environments/qa/hiera/common.json

```diff
@@ -134,7 +138,7 @@
     "hilary::config_activity_enabled": true,
     "hilary::config_previews_enabled": true,
 
-    "oaeservice::deps::package::nodejs::nodejs_version": "0.10.17-1chl1~precise1",
+    "oaeservice::deps::package::nodejs::nodejs_version": "node-v6.11.1-linux-x64",
```

Comment:
I think this is no longer needed as node is not being installed via deb package.